api/src/main/java/edu/wpi/first/shuffleboard/api/sources/recording/Recorder.java

```diff
@@ -54,7 +55,9 @@ private void saveToDisk() {
       return;
     }
     try {
-      String file = String.format(Storage.RECORDING_FILE_FORMAT, createTimestamp());
+      String date = dateFormatter.format(LocalDateTime.ofInstant(startTime, ZoneId.systemDefault()));
+      String time = timeFormatter.format(LocalDateTime.ofInstant(startTime, ZoneId.systemDefault()));
+      String file = Storage.RECORDING_FILE_FORMAT.replace("${date}", date).replace("${time}", time);
```

Comment:
I would prefer if this code (and the DateTimeFormatters) were in the Storage class, so that it has an api like File getRecordingFile(startTime)